src/OrbitClientProtos/capture_data.proto

```diff
@@ -110,7 +110,8 @@ message CaptureHeader {
 }
 
 message FrameTracksInfo {
-  repeated FunctionInfo frame_track_functions = 1;
+  reserved 1;
```

Comment:
what happens is that when user loads old capture they will not see frame tracks... but I would argue this is minor inconvenience and we shouldn't refuse to load a capture because of that.

Reply:
Makes sense.